### PR TITLE
SAAS-73 Reset telemetry UUID on telemetry disabling

### PR DESCRIPTION
**Pull request by artemgavrilov**
- https://jira.percona.com/browse/SAAS-73

FB: https://github.com/Percona-Lab/pmm-submodules/pull/851

### CODE REVIEW

**Comment by AlekSi** [comment]
For legal/privacy reasons.

File: models/settings_helpers_test.go [hunk 1]
```diff
@@ -218,5 +218,27 @@ func TestSettings(t *testing.T) {
...
+
+			ns, err = models.GetSettings(sqlDB)
+			require.NoError(t, err)
+			assert.Equal(t, uuid, ns.Telemetry.UUID)
+
+			ns, err = models.UpdateSettings(sqlDB, &models.ChangeSettingsParams{
+				DisableTelemetry: true,
+			})
+			require.NoError(t, err)
+			assert.Equal(t, "", ns.Telemetry.UUID)
```
Comment (AlekSi) [inline models/settings_helpers_test.go:241]: assert.Empty

**Comment by AlekSi** [comment]
@BupycHuk Please merge it once tree is open

File: models/settings_helpers_test.go (hunk 1 above)
Comment (artemgavrilov) [reply models/settings_helpers_test.go]: Fixed

**Pull request merged**